src/Data/Audience.elm

```diff
@@ -34,7 +36,30 @@ type alias Audience =
     , folder : Maybe Int
     }
 
+{-| Decoder for an Audience
+-}
+decodeAudience : JD.Decoder Audience
+decodeAudience = JD.map4
+    Audience
+        (JD.field "id" JD.int)
+        (JD.field "name" JD.string)
+        (JD.field "type" decodeAudienceType)
+        (JD.field "folder" (JD.maybe JD.int))
 
+{-| Decoder for AudienceType
+-}
+decodeAudienceType : JD.Decoder AudienceType
+decodeAudienceType = JD.string |> JD.andThen (\str ->
+    case str of
+        "curated" -> JD.succeed Curated
+        "shared" -> JD.succeed Shared
+        "user" -> JD.succeed Authored
+        sthElse -> JD.fail <| "Unknown AudienceType: " ++ sthElse)
+
+{-| Decoder for List of Audiences
+-}
```

Comment:
This kind of comment is kinda redundant. 
You can just rename `decodeAudiences` to `audiencesDecoder` and than you do not need comment at all :)

Reply:
That is a good point :) thank you.